src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -32,6 +34,16 @@ $(function () {
     renderAffix();
   }
 
+  // Format titles to break words
+  function formatTitles() {
+    var headers = $("article h1, article h2, article h3, article h4");
```

Comment:
add a new class break-text and apply it for xref class

---
In reply to: [118621480](https://github.com/dotnet/docfx/pull/1695#discussion_r118621480) [](ancestors = 118621480)